config/config.example.toml

```diff
@@ -10,7 +10,7 @@ host = "127.0.0.1"
 # For more details: https://actix.rs/docs/server/#graceful-shutdown
 shutdown_timeout = 30
 # HTTP Request body limit. Defaults to 16kB
-request_body_limit = 16_384
+request_body_limit = 32_768
```

Comment:
Wouldn't you need to update this in `development.toml` instead, if you need this value to be picked up by the test runner?